From 3f2d0cb56fc55a6423b824dabc3709132f57498c Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 6 Apr 2016 08:55:33 +0800
Subject: [PATCH] ARM64: config: enable the REGULATOR_PWM for rockchip

That's useful for every PWM controlled to adjust the voltage
regulators.

In the moment. We make savedefconfig to cleanup the rockchuip_cros_defconfig.

Change-Id: I33d68d6cd48310b2da0ea2c3331380e71fc51eee
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 6026fa6bc2f7..17cb0baf6daf 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -331,6 +331,7 @@ CONFIG_REGULATOR_ACT8865=y
 CONFIG_REGULATOR_FAN53555=y
 CONFIG_REGULATOR_GPIO=y
 CONFIG_REGULATOR_MP8865=y
+CONFIG_REGULATOR_PWM=y
 CONFIG_REGULATOR_RK808=y
 CONFIG_REGULATOR_RK818=y
 CONFIG_MEDIA_SUPPORT=y
@@ -357,13 +358,11 @@ CONFIG_MALI_DEBUG_SHADER_SPLIT_FS=y
 CONFIG_MALI_PLATFORM_THIRDPARTY=y
 CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
-
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 # CONFIG_BACKLIGHT_GENERIC is not set
 CONFIG_BACKLIGHT_PWM=y
 CONFIG_FB_ROCKCHIP=y
-
 CONFIG_LCD_MIPI=y
 CONFIG_RK_TRSM=y
 CONFIG_RK31XX_LVDS=y
-- 
2.35.3

